src/components/navbar.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import './bundle.css';
+
+const awsTopContributorsURL = '';
+const awsTiersURL = 'http://3.15.215.14:3101/';
+const awsDescriptionURL = 'http://54.193.6.166:3663/';
+
+class Bundle extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bundleId: parseInt(this.props.match.params.bundleId),
+      countdown: 1050023,
+      tierCosts: {
+        tier1: 1,
+        tier2: 3,
+        tier3: ((Math.floor(this.props.match.params.bundleId / 25) + 1) * 5),
+      }
+    }
+
+    this.getCountdown = this.getCountdown.bind(this);
+    this.getAverageCost = this.getAverageCost.bind(this);
+    this.getTopCost = this.getTopCost.bind(this);
+
+    this.getCountdown;
+    this.getAverageCost;
+    this.getTopCost;
```

Comment:
Is this a redundancy or WIP? I am uncertain how to read this (fully acknowledging that that could be on me and not an issue with your code!)

Reply:
This is ensuring that each of these get called during the constructor before render.